netebpfext/net_ebpf_ext.c

```diff
@@ -23,6 +23,7 @@
 #include "net_ebpf_ext_sock_ops.h"
 #include "net_ebpf_ext_xdp.h"
 
+#define XMSTO100NS(x) ((x)*10000)
```

Comment:
```suggestion
#define MSTO100NS(x) ((x)*10000)
```
Recommend removing X which doesn't mean anything in the define name.